### ISSUE TITLE
[Suggestion] Make a simple step by step installation tutorial

### ISSUE THREAD

**Issue by rafaelgdp** [godotengine/godot-monodevelop-addin#2]
See, I started using C# not so long ago and I found the language very promising. When I saw this plugin, I found its features really cool. However, I'm not sure how I should go about using it on my project. Is it just a "drag those files and drop" into the project folder? Or is there another approach like installing it to the IDE of choice like an "extension" somehow?

**Comment by ricardoalcantara** [comment]
@rafaelgdp This plugin lacks some documentation, but there is only one guy doing all the hard work about C#, anyway.
1 - Download the file from the release link
2 - Follow this steps https://www.monodevelop.com/documentation/installing-add-ins/

**Comment by rafaelgdp** [comment]
> @rafaelgdp This plugin lacks some documentation, but there is only one guy doing all the hard work about C#, anyway.
> 1 - Download the file from the release link
> 2 - Follow this steps https://www.monodevelop.com/documentation/installing-add-ins/

Thanks for the reply, @ricardoalcantara ! So, I went to the website and installed Visual Studio for Mac (that's what the monodevelop website suggests, as VS is built on top of monodevelop). The procedure to install an add-in changed slightly. It is now called "extension". I managed to click on install extension from file and the following error message appeared:
<img width="425" alt="Screen Shot 2020-02-12 at 15 09 32" src="https://user-images.githubusercontent.com/9672536/74363643-b9308d00-4da9-11ea-9e4d-36f9e8236cf1.png">
It actually installs, but when I try to use it, it says it is not possible due to incompatible monodevelop versions (mine is newer).

<img width="830" alt="Screen Shot 2020-02-12 at 15 10 19" src="https://user-images.githubusercontent.com/9672536/74363759-fac13800-4da9-11ea-9e48-0b9cb7bf7355.png">

How could I safely downgrade monodevelop version?

**Comment by ricardoalcantara** [comment]
Just to not to leave you without an answer which I don't have one =/. I don't know what to do, but I would not let it me down, this pluing just let you press Play and run the game xD. You can still code in VS ou VSCode (the one I use) and press play inside the Engine (Godot). If you want to debug you can still attach the process.

**Comment by rafaelgdp** [comment]
> Just to not to leave you without an answer which I don't have one =/. I don't know what to do, but I would not let it me down, this pluing just let you press Play and run the game xD. You can still code in VS ou VSCode (the one I use) and press play inside the Engine (Godot). If you want to debug you can still attach the process.

Yeah. I actually had configured those things in VS Code myself with custom tasks, but since I'm new to C#, I was not able to successfully configure a build task for the Godot C# project. I managed to fire a Godot instance of the game from VSCode by configuring a task that runs the Godot executable in the project working directory. But on the Monodevelop/VS side, I think I could solve this version compatibility issue from this addin by building my own version of the .mpack file from source. Looking over the source code, I could not find a reference to this minimum compatible version. It must be embedded to the mpack file upon compiling. So that might solve this issue!

**Comment by neikeq** [comment]
I will have a look at the issue with newer versions.

**Comment by neikeq** [comment]
I've created a new [release](https://github.com/godotengine/godot-monodevelop-addin/releases) which includes a package for MonoDevelop and another one for Visual Studio for Mac. Let me know if this one works for you.

**NOTE:** This new release of the extension is only compatible with Godot 3.2.2 or greater.

**Comment by code5717** [comment]
will it work for Linux ?

**Comment by ricardoalcantara** [comment]
@Airbus5717 Yes!

**Comment by SjVer** [comment]
I've encountered a similar issue, but none of the releases work for me. Any idea on why or how to fix it?
